packages/hoppscotch-cli/src/options/test/env.ts

```diff
@@ -42,6 +42,7 @@ export async function parseEnvsData(path: string) {
 
   if (HoppEnvKeyPairResult.success) {
     for (const [key, value] of Object.entries(HoppEnvKeyPairResult.data)) {
+      // PR-COMMENT: type error
```

Comment:
We can resolve this by making the type account for the [legacy export format](https://docs.hoppscotch.io/documentation/clients/cli#2-legacy-export-format) that the CLI supports.

```ts
const envPairs: Array<HoppEnvPair | Record<string, string>> = [];
```

https://github.com/hoppscotch/hoppscotch/blob/faab1d20fde9a6be660db40fc73dcf28f9038008/packages/hoppscotch-cli/src/options/test/env.ts#L20